lib/cirro_io/client/response_debugging_middleware.rb

```diff
@@ -3,7 +3,7 @@ module CirroIO
   module Client
     class ResponseDebuggingMiddleware < Faraday::Response::Middleware
       def on_complete(env)
-        # binding.pry
+        binding.pry # rubocop:disable Lint/Debugger
```

Comment:
Is this a leftover artifact?

Reply:
Actually i added this middleware for writing tests. https://github.com/test-IO/cirro-ruby-client/blob/master/lib/cirro_io/client/base.rb#L19
So the idea is when I want to write test, i enable this middleware and run the test against actual API. Here in pry i get the response and then put that response in json files below
https://github.com/test-IO/cirro-ruby-client/tree/master/spec/fixtures

Reply:
Maybe its worth to checkup how does vcr gem behaves now. In past we had lot of trouble with maintaining cassettes